### DISCUSSION TITLE
New Tagging system (23.3.0)

### DISCUSSION THREAD

**Discussion by Thompson-ongithub** [mytourbook/mytourbook#1014] [Q&A]
Hello

I just tried the new version of MTB (23.3.0 on Linux) and whilst I was very happy about the use of the embedded browser when searching for tours, it was the revamped tagging system that makes me a bit unhappy.
The reason is that in my setting, it's not only equipment that I'm tagging, but also people I do rides with, so the new tags with photos and hours and miles provides information I don't need (and it takes up way more space) and sometimes even more information, since in difference to the "Tagged Tours"-View where only the selected tour type is calculated, this new feature just adds up all the information of all the tour types (ex. cycling and running), which might be ok when looking at running shoes, but is of no help if I want to know how often / long I've been running with a buddy of mine. 
Is there a way to turn that of (I tried but haven't found one)

Oh, and since I'm talking about tagging: is there a setting to have the tags displayed not in an alphabetical order but grouped by category and within that group alphabetical (should I make a feature request for this)

sorry for moaning about a new feature (otherwise MTB is great!)

**Comment by wolfgang-ch** [comment]
>  Is there a way to turn that of (I tried but haven't found one)

Not yet and I agree with you that this info should be optional, customizable

> Oh, and since I'm talking about tagging: is there a setting to have the tags displayed not in an alphabetical order but grouped by category and within that group alphabetical (should I make a feature request for this)

Not yet, how should the categories be sorted?

> (should I make a feature request for this)

Yes

**Reply by wolfgang-ch** [reply]
I've created an issue for the optional tag info feature https://github.com/mytourbook/mytourbook/issues/1015

**Reply by Thompson-ongithub** [reply]
Thanks
